### DISCUSSION TITLE
ARM64 Package

### DISCUSSION THREAD

**Discussion by dev-nextprogress** [emgucv/emgucv#511] [Q&A]
Hi,
I am evaluating Emgu.CV for .NET Core applications that will execute in an ARM64 runtime architecture, like an NVIDIA Xavier. I am finding references to Raspberry PI but nothing about general Linux ARM64. It the Raspberry PI compatible or what would I need for the Linux ARM64 runtime ?
Thanks

**Comment by nvsnkv** [comment]
Hi! I was able to build emgucv from sources on Raspberry Pi OS 64 bit and was able to consume at least mjpeg streams from IP camera.
I followed [this guide](https://www.emgu.com/wiki/index.php/Download_And_Installation#Raspberry_Pi_OS) with a few tweaks:
1. libjasper-dev is no longer maintained and due to that cannot be installed using apt. I excluded this package from `apt_install_dependency` and added additional __cmake__ flag to `cmake_configure` to build Jasper from sources (`-DBUILD_JASPER:BOOL=TRUE \` next to `-DBUILD_PNG:BOOL=TRUE`). I'll prepare a pull request for that once I get all my issues resolved.
2. After successful compilation  (couple of hours :) ) I copied the binaries from `/libs` and `/libs/arm64` folders in the repository root to a separate location (just for my convinience) and added this location to `LD_LIBRARY_PATH` env variable, as suggested in the guide.
3. Then I compiled .NET app for arm64 and just started it

For some reason I was not able to use builtin camera, so I will raise another discussion to find out what did I do wrong :)

**Reply by nvsnkv** [reply]
I don't think Raspberry Pi is special there - any platform in which you can compile emgucv from sources should work.
I also was able to build it for Ubuntu 20.04, as an example
